README.md

```diff
@@ -153,6 +153,14 @@ usage is clear you can use `U`, `V` etc. for any following generic.
 If the usage is not obvious, you should use a more descriptive name. The same naming rules as for
 classes will apply then.
 
+### Prefixing
+
+#### Types and Interfaces
+
+Do not prefix types with a `T` or interfaces with an `I`. For example `new Array<TData>();` there
+is no need to prefix them with a `T`. It's even better to give the type a descriptive name like
+`new Array<Person>();`.
+
```

Comment:
``````suggestion
:thumbsup: Example of **correct** type names:

```interface Person {
  firstName: string;
  lastName: string;
};

type Person = {
  firstName: string;
  lastName: string;
};
```

:thumbsdown: Example of **incorrect** type names:

```interface IPerson {
  firstName: string;
  lastName: string;
};

type TPerson = {
  firstName: string;
  lastName: string;
};
```
``````